lib/ukboot/boot.c

```diff
@@ -238,8 +238,7 @@ static struct uk_alloc *heap_init()
 extern char **environ;
 #endif /* CONFIG_LIBPOSIX_ENVIRON */
 
-/* defined in <uk/plat.h> */
-void ukplat_entry(void)
+void uk_boot_entry(void)
```

Comment:
There is a `_` too much in the commit title in `uk_plat_entry` :)

Reply:
Actually `_ukplat_entry` is correct, I am referring to the platform entry, not the boot entry: https://github.com/unikraft/unikraft/blob/staging/plat/kvm/x86/setup.c#L72 🙂

Reply:
What I meant is the title of this commit that does this change:
```
lib/ukboot: Rename uk_plat_entry() to uk_boot_entry() 
```
Shouldn't it be?:
```
lib/ukboot: Rename ukplat_entry() to uk_boot_entry() 
```

Reply:
That one yes 👍🏼